src/components/authorization/AuthForm.css

```diff
@@ -32,3 +32,6 @@
 .wrapper-username label {
   margin-bottom: 0.5rem;
 }
+.btn-show-password {
+  cursor: pointer;
```

Comment:
have something like cursor pointer in a class and then reuse it I know this is small thing but still it can make it an iota more modular.

Reply:
Got it. Thank you for the suggestion.